icons/bowling-alley.svg

```diff
@@ -1,4 +1,4 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<svg id="bowling-alley" xmlns="http://www.w3.org/2000/svg" width="15" height="15" viewBox="0 0 15 15">
-  <path id="icon_concepts:_sports_leisure" d="M8.83,5.39a11.0127,11.0127,0,0,0-.96,4.28,11.4686,11.4686,0,0,0,.58,3.69A4.9424,4.9424,0,0,1,6,14,5,5,0,1,1,8.98,4.99C8.94,5.12,8.88,5.25,8.83,5.39ZM4.12,8.5a.62.62,0,1,0-.62.62h0a.6105.6105,0,0,0,.62-.6009C4.12,8.5128,4.12,8.5064,4.12,8.5ZM4.5,7.12a.62.62,0,1,0-.62-.62h0a.6106.6106,0,0,0,.6011.62ZM6.12,8.5a.62.62,0,1,0-.62.62h0a.6105.6105,0,0,0,.62-.6009C6.12,8.5128,6.12,8.5064,6.12,8.5Zm4.8022-5.3341c0-.7222-.1444-.7222-.1444-1.4444a.7222.7222,0,0,1,1.4444-.0007v.0007c0,.7222-.1444.7222-.1444,1.4444,0,2.1667,1.5528,3.6168,1.5528,6.5,0,2.1667-.6861,4.3333-1.4083,4.3333H10.7778c-.7222,0-1.4083-2.1667-1.4083-4.3333C9.3694,6.7826,10.9222,5.3325,10.9222,3.1659Z"/>
+<svg width="15" height="15" viewBox="0 0 15 15" xmlns="http://www.w3.org/2000/svg" id="bowling-alley">
+  <path d="M10.9222 3.16585C10.9222 2.80475 10.8861 2.6242 10.85 2.44365 10.8139 2.2631 10.7778 2.08255 10.7778 1.72145 10.7777 1.52991 10.8537 1.34618 10.9891 1.21068 11.1245 1.07517 11.3081.998995 11.4997.998901 11.6912.998809 11.8749 1.07481 12.0104 1.21018 12.1459 1.34555 12.2221 1.52921 12.2222 1.72075 12.2222 2.08185 12.1861 2.26258 12.15 2.4433 12.1139 2.62403 12.0778 2.80475 12.0778 3.16585 12.0778 4.17196 12.4126 5.02355 12.7713 5.93586 13.1851 6.98826 13.6306 8.12146 13.6306 9.66585 13.6306 11.8326 12.9445 13.9992 12.2223 13.9992H10.7778C10.0556 13.9992 9.36951 11.8325 9.36951 9.66585 9.36945 8.12135 9.815 6.98809 10.2288 5.93566 10.5874 5.0234 10.9222 4.17186 10.9222 3.16585ZM8.91959 6.57474C8.62376 7.4299 8.36947 8.43163 8.36951 9.66589 8.36951 10.7024 8.51281 11.7479 8.75007 12.6124 7.93094 13.4675 6.77765 14 5.5 14 3.01472 14 1 11.9853 1 9.5 1 7.01472 3.01472 5 5.5 5 6.86846 5 8.09426 5.61084 8.91959 6.57474ZM3.4735 8.40932C3.24509 8.45475 3.05636 8.64348 3.01093 8.87188 2.9396 9.2305 3.22719 9.57548 3.58836 9.57548 3.82033 9.57912 4.03738 9.43895 4.1295 9.22602 4.20836 9.04375 4.18824 8.82483 4.07787 8.65966 3.94849 8.46602 3.70191 8.36388 3.4735 8.40932ZM5.11536 6.97273C5.04404 6.61419 4.65037 6.40377 4.31263 6.54367 4.01809 6.66567 3.87287 7.01479 3.98989 7.31014 4.07867 7.53425 4.29863 7.67633 4.53793 7.67633 4.90349 7.67633 5.18667 7.33127 5.11536 6.97273ZM5.37265 8.40932C5.14424 8.45475 4.95552 8.64348 4.91008 8.87188 4.88737 8.98609 4.89903 9.10446 4.94359 9.21204 5.06558 9.50656 5.41468 9.65178 5.71003 9.53483 5.8516 9.47877 5.9682 9.36577 6.02866 9.22602 6.15229 8.94026 6.00503 8.56386 5.71281 8.44282 5.60523 8.39826 5.48686 8.3866 5.37265 8.40932Z"/>
 </svg>
```

Comment:
Should the finger hole circles continue to be as large as they are in the current design? They seem like important indicators. Also seeing some additional anchors around the finger holes.

Reply:
Thank you, I totally agree. I returned the old holes and deleted unnecessary anchors